x-pack/typings/hapi.d.ts

```diff
@@ -22,6 +22,6 @@ declare module 'hapi' {
     security?: SecurityPlugin;
     actions?: ActionsPlugin;
     alerting?: AlertingPlugin;
-    task_manager?: LegacyTaskManagerApi;
+    task_manager?: TaskManager;
```

Comment:
These extend the hapi server so that they can be passed in on the context.
I decided to leave it until that part of core is removed entirely which I expect platform will don once everyone has migrated.

Reply:
I'll make sure there's an issue for the remaining bits